inventory_service/src/main.py

```diff
@@ -46,3 +46,4 @@ def delete_container(container_name):
     return items
 if __name__ == '__main__':
     app.run(debug=True,host = '127.0.0.1',port = 4000)
+    
```

Comment:
route names can be improved
![image](https://user-images.githubusercontent.com/70574489/217153527-014fff61-07ea-4089-bae2-69585e4f938d.png)

Just naming it as 'item' should be good. The api method will tell which action to perform